src/test/java/redis/clients/jedis/JedisPooledTest.java

```diff
@@ -231,20 +231,23 @@ public void cleanUp() {
       }
     };
 
+    // TODO: do it without the help of pool config; from Connection constructor? (configurable) force ping?
```

Comment:
> TODO: do it without the help of pool config; from Connection constructor? (configurable) force ping?

I don't think we need to do much about it.

If the user's JedisPoolConfig does not configure the correct password, it is normal to return NOAUTH in subsequent requests.